vision/getting-started/image_editing.ipynb

```diff
@@ -1,710 +1,1247 @@
 {
```

Comment:
<p>The notebook should show an example with an image and an actual prompt. Otherwise, as a user, I'd have to waste several minutes finding my own image and testing the feature, and if it doesn't work well, I might not know if it's because I did something wrong or if the feature didn't work.</p><p>It's easier to demonstrate best practices for users here but giving an actual sample image and actually do the inpainting-insert.</p><p> </p><p>We should have a sample images and actual sample prompts used throughout the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Planning on adding visual samples in the next change